app/views/submissions/_submission_evaluation.html.erb

```diff
@@ -0,0 +1,8 @@
+<h2>Submission Evaluation</h2>
+<% if @submission.phase.evaluation_form.nil? %>
+  <%= render 'shared/alert_error', alert_heading: t('alerts.no_evaluation_form.heading'), alert_text: t('alerts.no_evaluation_form.text') %>
+<% else %>
+    <%= render partial: 'assigned_evaluators' %>
+    <%= render partial: 'available_evaluators' %>
+    <%= link_to "Manage Evaluators", phase_evaluators_path(@submission.phase), class: "float-right margin-y-1"  %>
```

Comment:
I noticed this link overlaps with the footer on mobile size, probably better to use flexbox than floats?
![Screenshot 2025-01-07 at 11 12 57 AM](https://github.com/user-attachments/assets/a7d42096-3266-422a-82a3-e0c2b0c38f38)